stdlib/public/core/Collection.swift

```diff
@@ -387,12 +387,11 @@ public protocol Collection: Sequence {
   /// Returns an iterator over the elements of the collection.
   override __consuming func makeIterator() -> Iterator
 
-  /// A sequence that represents a contiguous subrange of the collection's
-  /// elements.
+  /// A collection whose values represent contiguous subranges of this
+  /// collection's elements.
   ///
-  /// This associated type appears as a requirement in the `Sequence`
-  /// protocol, but it is restated here with stricter constraints. In a
-  /// collection, the subsequence should also conform to `Collection`.
+  /// The subsequence shares the same indices for its elements as the
+  /// original collection.
```

Comment:
You could use words closer to what's in "Slices Share Indices" earlier in the document; something like "the collection and its subsequences share the same indices". Should this be the place that ties "SubSequence" to the "Slice" concept? (I'm not finding whether the link is made anywhere).

Reply:
I added a passage calling out the default value!